packages/ts/react-form/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@hilla/react-form",
-  "version": "2.3.0-beta1",
+  "version": "2.3.0-beta.2",
```

seems not related, let's revert:

```suggestion
  "version": "2.3.0-beta1",
```